### PR TITLE
Implemented code to show status badge on history segment of pipeline page(#257kwwn)

### PR DESCRIPTION
**Pull request by azkyakhan**
### Related Issues

Closes #

### Screenshots of Visual Changes before/after (If There Are Any)

**IMPORTANT NOTICE** - Remember to add changelog entry

### Contribution and Currently Important Rules Acceptance

- [ ] I read and followed [contribution rules](https://github.com/hotwax/job-manager#contribution-guideline)

### CODE REVIEW

**Comment by azkyakhan** [comment]
status-failed

Since there is no data coming in finished status card it is looking like this :

status-finished

**Comment by azkyakhan** [comment]
localhost_8100_pipeline

**Comment by azkyakhan** [comment]
status

File: src/store/modules/user/getters.ts [hunk 1]
```diff
@@ -3,26 +3,29 @@ import UserState from './UserState'
...
-    getInstanceUrl (state) {
-        return state.instanceUrl;
-    },
-    getShopifyConfigId (state) {
-        return state.shopifyConfig;
-    },
-    getCurrentEComStore(state) {
-        return state.currentEComStore
-    },
+  isAuthenticated (state) {
```
Comment (adityasharma7) [inline src/store/modules/user/getters.ts:6]: Unwanted indentation changes.

File: src/store/modules/user/index.ts [hunk 2]
```diff
@@ -6,20 +6,21 @@ import UserState from './UserState'
 import RootState from '@/store/RootState'
 
 const userModule: Module<UserState, RootState> = {
-    namespaced: true,
```
Comment (adityasharma7) [inline src/store/modules/user/index.ts:9]: Unwanted indentation changes

File: src/store/modules/user/UserState.ts [hunk 3]
```diff
@@ -4,4 +4,5 @@ export default interface UserState {
     currentEComStore: object;
     instanceUrl: string;
     shopifyConfig: object;
+    statusDesc: any;
```
Comment (adityasharma7) [inline src/store/modules/user/UserState.ts:7]: I think it should part of util

File: src/store/modules/user/mutations.ts [hunk 4]
```diff
@@ -3,24 +3,29 @@ import UserState from './UserState'
...
-    [types.USER_INSTANCE_URL_UPDATED] (state, payload) {
-        state.instanceUrl = payload;
-    },
-    [types.USER_SHOPIFY_CONFIG_UPDATED] (state, payload) {
-        state.shopifyConfig = payload;
-    },
-    [types.USER_CURRENT_ECOM_STORE_UPDATED] (state, payload) {
-        state.currentEComStore = payload;
-    },
+  [types.USER_TOKEN_CHANGED] (state, payload) {
```
Comment (adityasharma7) [inline src/store/modules/user/mutations.ts:6]: Unwanted indentation changes

File: src/views/Pipeline.vue [hunk 5]
```diff
@@ -68,7 +68,10 @@
                 <ion-card-subtitle class="overline">{{ job.parentJobId }}</ion-card-subtitle>
                 <ion-card-title>{{ getEnumName(job.systemJobEnumId) }}</ion-card-title>
               </div>
-              <ion-badge v-if="job.runTime" color="dark">{{ timeTillJob(job.runTime)}}</ion-badge>
+              <div>
+                <ion-badge v-if="job.runTime" color="dark">{{ timeTillJob(job.runTime)}}</ion-badge>
+                <ion-badge v-if="job.statusId" :color="job.statusId === 'SERVICE_FINISHED' ? 'success' : 'danger'">{{ getStatusDesc(job.statusId) }}</ion-badge>
```
Comment (adityasharma7) [inline src/views/Pipeline.vue:73]: I think we should merge the status description in original job instead of using getter in markup

**Review by adityasharma7 — CHANGES_REQUESTED** [review]

File: src/store/modules/user/getters.ts (hunk 1 above)
Comment (azkyakhan) [reply src/store/modules/user/getters.ts]: Ok sir I have removed unwanted indentation.

File: src/store/modules/user/index.ts (hunk 2 above)
Comment (azkyakhan) [reply src/store/modules/user/index.ts]: Ok sir I have removed unwanted indentation.

File: src/store/modules/user/UserState.ts (hunk 3 above)
Comment (azkyakhan) [reply src/store/modules/user/UserState.ts:7]: I have created a new module util and stored status related information in it

File: src/store/modules/user/mutations.ts (hunk 4 above)
Comment (azkyakhan) [reply src/store/modules/user/mutations.ts]: Ok sir I have removed unwanted indentation.

File: src/views/Pipeline.vue (hunk 5 above)
Comment (azkyakhan) [reply src/views/Pipeline.vue]: Ok sir.

**Review by adityasharma7 — APPROVED** [review]

**Pull request merged**